arches/app/etl_modules/base_data_editor.py

```diff
@@ -188,10 +192,29 @@ def get_resourceids_from_search_url(self, search_url):
         return [result["_source"]["resourceinstanceid"] for result in results]
 
     def validate(self, request):
-        return {"success": True, "data": {}}
+        raise NotImplementedError
+
+    def validate_inputs(self, request):
+        raise NotImplementedError
 
 
 class BulkStringEditor(BaseBulkEditor):
+    def validate(self, request):
+        return {"success": True, "data": {}}
+
+    def validate_inputs(self, request):
+        operation = request.POST.get("operation", None)
+        required_inputs = [
+            "graph_id", "node_id","operation","language_code"
+        ]
+        if operation == "replace":
+            required_inputs = required_inputs + ["old_text","new_text"]
+
```

Comment:
It would be nice to add a lookup to get a localized string that matches the input label. e.g.
```
{
"language_code": _('Language'),
"graph_id": "Resource Model": _('graph_id'),
...
}
```